content/pages/joinus.md

```diff
@@ -0,0 +1,103 @@
+---
+title: شارك معنا 
+slug: joinus
+template: page
+---
+
+## شارك معنا
+نرحب بالجميع للانضمام الى فريق شارك تك ونشر المقالات في مجالات التقنية المختلفة. من أهم أهداف هذه المدونة هو إثراء المحتوى العربي بالشبكة العنكبوتية الانترنت ، فلذلك تم إستخدام تصميم مناسب للغة العربية  من حيث التنسيق وتجاه النصوص بالمدونة. 
+
+من الممكن في المستقبل القريب إنشاء موقع آخر مختص بنشر المقالات باللغة الإنجليزية في حال وجدنا رغبة كبيرة من أعضاء الفريق.
+
+## مميزات المدونة
+تم إنشاء الموقع على منصة `github` وذلك لأسباب عدة من ضمنها نشر شفرة `code` الخاص بالموقع وذلك لدعم المصدر المفتوح بالإضافة الى حفظ حقوق الأعضاء المشاركين وإعطائهم صلاحية نقل مقالاتهم بكل سهولة في حال رغبو بذلك.
+
+### بعض المميزات
+- جميع المواضيع المتعلقة بالتقنية مرحب بها.
+- المقالات ملك للناشر وله الحق التصرف بالمواضيع الخاصة فيه من ناحية حذف أو نقل مقالاته الخاصة فيه بالمستقبل.
+-  دعم ومساعدة الجميع لنقل ونشر المعرفة.
+- نعمل على رفع مستوى جودة المقال حيث يتم مراجعة المقال و تقديم الملاحظات قبل نشر المقال.
+
+## الدليل الإرشادي للانضمام والمشاركة
+قد يبدو الدليل والمشاركة في المدونة معقد قليلا خصوصا لمن ليس لديه خلفيه برمجية، ولكن نؤكد لكم إنه بعد التجربة المتكررة تكون المشاركه سهله جدا وتمنح الكاتب التحكم الكامل بالموضوع.
+
+### الخطوة الأولى
+أول خطوة للمشاركة في المدونة هو التسجيل في موقع `github`.
+رابط الموقع: [github](https://github.com/)
+
+### الخطوة الثانية 
+بعد التسجيل في موقع `github`، يتم إنشاء نسخة من المستودع `repo`
+ الخاص بموقع شارك تك عن طريق مايعرف ب fork.
+
+نذهب لرابط مستودع شارك تك([رابط المستودع](https://github.com/sharektech/sharektech.com)) ثم نقوم بعمل `fork` كما هو موضح بالصورة:
+![fork sharektech repo](../images/joinus/sharektech-github-link.png "fork sharektech repo")
+ الهدف من هذه النسخة هو عمل مقالات الناشر في مستودع خاص به للمراجعة قبل دمجها مع مقالات الموقع.
+
+### الخطوة الثالثة
```

Comment:
So from here i am not sure if everyone who sees this will be excited to contribute since it's a bit advanced to manage and create ssh keys. We can refer to the browser editor within github it's self. They can fork and just start adding files that they need etc.. . Up for discussion

Reply:
I agree , we will post it for now , then find a cleaner way to present the join us page.